client/src/image.jsx

```diff
@@ -1,19 +1,158 @@
+import styled from 'styled-components'
 import React from 'react';
+import StarRatings from 'react-star-ratings';
 
-const imgStyle = {
-  height: '200px',
-  width: '125px'
-}
+const BookCover = styled.img`
+  width: 150px;
+  height: 200px;
+`;
+
+const PreviewWrapper = styled.div`
+  padding: 10px;
+  text-align: center;
+`
+
+const WantToRead = styled.button`
+  width: 110px;
+  background-color: #409D69;
+  color: #fff;
+  font-size: inherit;
+  -webkit-font-smoothing: antialiased;
+  height: 28px;
+  padding: 6px 0 6px 8px;
+  box-sizing: border-box;
+  border: 0;
+  cursor: pointer;
+  display: block;
+  font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  margin: auto;
+  text-align: inherit;
+  border-bottom-left-radius: 3px;
+  border-top-left-radius: 3px;
+  border-style: solid;
+`
+
+const ChoiceButton = styled.div`
+  border-width: 0px;
+  width: 120px;
+  font-size: 13px;
+  box-sizing: content-box;
+  font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: bold;
+  text-align: left;
+  word-wrap: normal;
+  white-space: nowrap;
+  padding: 0;
+  margin: 10px 0 0px 10px;
+  font-weight: bold;
+  display: flex;
+`
+
+const Dropdown = styled.button`
+  box-sizing: border-box;
+  background-color: #409D69;
+  color: #fff;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: 8px 4px;
+  border: 0;
+  cursor: pointer;
+  display: block;
+  font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: inherit;
+  font-style: inherit;
+  font-weight: inherit;
```

Comment:
Aren't such properties inherited by default?

Reply:
@MuhammadJamaluddin good catch - I just cleaned up some extraneous CSS (`size` needed inheriting, but not `style` / `weight` in this instance)